tools/layer-zero-example/test/htsAdapterTests.js

```diff
@@ -0,0 +1,139 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Hardhat Example
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const hre = require('hardhat');
+const { ethers } = hre;
+const { Options, addressToBytes32 } = require('@layerzerolabs/lz-v2-utilities');
+const { expect } = require('chai');
+
+const HEDERA_EID = 40285;
+const BSC_EID = 40102;
+const receiverAddress = '0xF51c7a9407217911d74e91642dbC58F18E51Deac';
+const amount = '100';
+
+describe('HTSAdapterTests', function() {
+  it('@hedera @fund-and-approve transfer to adapter', async () => {
```

Comment:
Similar to other comment can you log all the state changing transaction hashes for all the tests to make it easy to tell the E2E story on a given run